src/main.py

```diff
@@ -25,16 +26,19 @@
 # SOFTWARE.
 
 if __name__ == '__main__':
-    usr = ''
-
-    if len(argv) > 2:
-        exit('usage: python3 ' + argv[0] + ' username')
-    elif len(argv) == 1:
-        usr = input('Profile\'s name: ')
-    else:
-        usr = argv[1]
+    parser = ArgumentParser(prog="gps.py")
+    parser.add_argument("profile_name", dest="user", nargs="?", type=str,
+        help="Your GitHub profile name")
+    args = parser.parse_args()
+    user = args.user
+    if user is None:
+        user = input("Profile's name: ")
+        
+    if len(user) == 0:
```

Comment:
wouldn't 
```python
if not user:
    # ...
```
do the same ?

Reply:
As opposed to `if user is None:`? Sure, that would technically do the same thing. I'll have that changed in just a moment.

Reply:
This has been changed in commit [56d62b0](https://github.com/aewens/gps/commit/56d62b0d1af1f766416c8b3ac2e44c9f1512cce6#diff-0c81b9c94c68ea34c497c1fa52daee56)